core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -867,6 +868,39 @@ class SocketServerTest {
     }
   }
 
+  @Test
+  def testExceptionInAcceptor(): Unit = {
+    val overrideNum = server.config.maxConnectionsPerIp + 1
+    val overrideProps = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 0)
+    overrideProps.put(KafkaConfig.MaxConnectionsPerIpOverridesProp, s"localhost:$overrideNum")
```

Comment:
I guess that we don't need this in the context of this test, do we?

Reply:
We need the `overrideProps` to create the `overrideServer`. From the other tests, I didn't figure out what could be a better configuration, so I just use the configuration in `testMaxConnectionsPerIpOverrides`. Or maybe we can try the configuration in `testConnectionRatePerIp`. Do you have any suggestion?

Reply:
Sorry, I was not clear here. Yes, we do need `overrideProps`. However, I think that we don't need to set `MaxConnectionsPerIpOverridesProp`, do we? If not, we could remove it and also remove `overrideNum`.

Reply:
You're right. I've pushed a new commit